src/cunumeric/cunumeric_c.h

```diff
@@ -185,6 +187,19 @@ enum CuNumericBounds {
   CUNUMERIC_MAX_TASKS   = 1048576,
 };
 
+// Match these to CuNumericTypeCodes in config.py
+enum CuNumericTypeCodes {
+  CUNUMERIC_TYPE_POINT1 = LEGION_TYPE_TOTAL + 1,
```

Comment:
It's not the contract that `LEGION_TYPE_TOTAL` is the total number of types in the core's type system, and it's about to change: https://github.com/nv-legate/legate.core/pull/199/files#diff-b0130510eeec3744cdd00fb2d06435229c70360346698827f0d4fb3553ebb67aR77 I think you should use `MAX_TYPE_NUMBER` instead.

Reply:
@magnatelee : thank you, I will change that.